### PR TITLE
fix: Replace BEARER_TOKEN constant to be a function

### PR DESCRIPTION
**Pull request by AmandaMisjuwar**
## Notion ticket link

[Access management table not loading on first visit](https://www.notion.so/uwblueprintexecs/Access-Management-Table-not-loading-on-first-visit-0ccaa7ab18504bb787071999442079c9)

## Implementation description
* The access management table was not loading on first load because the BEARER_TOKEN was `null` for some reason, we found that changing the `BEARER_TOKEN` constant to be a function fixed the issue, probably bc when the `BEARER_TOKEN` first gets its value upon page load, it doesn't update unless we refresh the page, but we want to get the access token value at the given moment

## Steps to test
1. Login
2. Go to access management page for the first time upon logging in
3. Data in the table should load without refreshing

## What should reviewers focus on?
* 

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

**Comment by AmandaMisjuwar** [comment]
Also, again I'm not able to run the linter so if somebody who's reviewing this can run `yarn fix` on the frontend that would be great :DD

**Review by dhmi19 — APPROVED** [review]
lgtm. I'll let @e-wai take a look too since this impacts all our frontend calls

File: frontend/src/constants/AuthConstants.ts
Comment (e-wai) [inline frontend/src/constants/AuthConstants.ts:5]: linter

**Review by e-wai — APPROVED** [review]
good catch, can we fix the return type of function linter error before merging?

**Comment by e-wai** [comment]
FWIW it's still very slow for me, will check on staging once this is merged

https://user-images.githubusercontent.com/40974372/204486567-1ad07fe8-a993-4b3e-83a9-c1b1c9d41cae.mov

**Pull request merged**
